src/IssueViz/Editor/SelectedIssueTagging/ThemeColors.cs

```diff
@@ -0,0 +1,70 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Windows.Media;
+using Microsoft.VisualStudio.PlatformUI;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Editor.SelectedIssueTagging
+{
+    internal static class ThemeColors
+    {
+        internal static readonly IThemeColors LightTheme = new LightThemeColors();
+        internal static readonly IThemeColors DarkTheme = new DarkThemeColors();
+
+        public static bool IsLightTheme(Brush textBrush)
+        {
+            var isLightTheme = true;
+
+            if (textBrush is SolidColorBrush solidColorBrush)
+            {
+                var isDarkText = solidColorBrush.Color.IsDark();
```

Comment:
IsDark/IsLight is a built-in functionality.
Basically, we need to set a contrast between the text and the highlight/adornment. So if the text font is dark, then we should display a light adornment and highlight. Otherwise, if the text is light, then we should display a dark adornment.